dora/core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -645,7 +635,8 @@ default PositionReader openPositionRead(AlluxioURI path) {
    * @param options options to associate with this operation
    * @return a {@link PositionReader} for the given path
    */
-  PositionReader openPositionRead(AlluxioURI path, OpenFilePOptions options);
+  PositionReader openPositionRead(AlluxioURI path, OpenFilePOptions options)
+      throws FileDoesNotExistException;
```

Comment:
This call implicitly relies on `getStatus()`, so it should throw the same exceptions as `getStatus()`. This change here should not be considered a behavior change. It's more like in the existing implementation we did not correctly declare the exception in the signature. 

I realized we need this change here because of the change in `MetadataCachingFileSystemTest`. `FileSystem.getStatus() throws FileDoesNotExistException` but that was incorrectly missed out in `UfsBaseFileSystem.getStatus()`.